ZocBuild.Database.Tests/Fakes/FakeDirectoryInfo.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using System.Security.AccessControl;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ZocBuild.Database.Tests.Fakes

```

Comment:
You can obviate the need for all these fakes by using the [System.IO.Abstractions.TestingHelpers](http://www.nuget.org/packages/System.IO.Abstractions.TestingHelpers/) package.